src/Renderer/ThreeExtended/GeoJSON2Three.js

```diff
@@ -141,10 +166,10 @@ const GeometryToVertices = {
         return result;
     },
 
-    multiPolygon(crsIn, crsOut, coordinates, filteringExtent) {
+    multiPolygon(crsIn, crsOut, coordinates, filteringExtent, options) {
         let result;
         for (const polygon of coordinates) {
-            const p = this.polygon(crsIn, crsOut, polygon, filteringExtent);
+            const p = this.polygon(crsIn, crsOut, polygon, options);
```

Comment:
polygon function signature is: `polygon(crsIn, crsOut, coordinates, filteringExtent, options)` so there's something wrong here.

Reply:
And this `options` object seems only to be used to store `buildExtent` or `toMesh`... cf my earlier comment: I don't think this is a good idea.